compiler-rt/lib/mpk_untrusted/mpk_fault_handler.cpp

```diff
@@ -24,12 +24,11 @@ void disableMPK(siginfo_t *si, void *arg);
 // MPK faults would be required).
 void segMPKHandle(int sig, siginfo_t *si, void *arg) {
   if (si->si_code != SEGV_PKUERR) {
-    REPORT("INFO : SegFault other than SEGV_PKUERR, handling with "
-           "default handler.\n");
+    REPORT("INFO : SegFault other than SEGV_PKUERR.\n");
     // SignalHandler was invoked from an error other than MPK violation.
     // Perform default action instead and return.
     if (!prevAction) {
-      REPORT("ERROR : prevAction is null, no previous handler to fall back to.\n");
+      REPORT("INFO : There is no previously saved action for SIGSEGV, handling with default signal handler.\n");
```

Comment:
General comment: your logging messages are too English like. Think about every compiler error you've ever read, or diagnostic from a tracing environment. They are terse, but informative when they are best. 

suggestion here:
```No existing SIGSEGV handler, falling back to default signal handler``` or something along these lines.

Reply:
Changed message as suggested.